Automattic/Gravatar-SDK-Android
# Quick Editor: Fix popup alignment above anchor

Closes #419 

### Description

The avatar contextual menu popup was previously centered on screen, a behavior inherited from the other popup menu for the "upload" button (`MediaPickerPopup`). 

I changed the `PickerPopup` so it's always horizontally centered on the anchor, thus removing a bunch of unneeded code (mainly parameters). The "Upload" button is centered on screen anyway, so the popup should also be centered. If for some reason we have the Gravatar bottomsheet appear on the side (tablet split-screen mode, side-by-side fragments? idk), I'd prefer that the popup appears above the button and not in the middle of the screen.


https://github.com/user-attachments/assets/a00e8da0-2b47-42c8-8a2a-4de3b6e936e5



### Testing Steps

1. Open the QE
2. Tap the options button (...) on an avatar on the left
3. See that the popup appears on the left above the avatar
2. Tap the options button (...) on an avatar on the right
3. See that the popup appears on the right above the avatar
2. Tap the options button (...) on an avatar on the center
3. See that the popup appears centered above the avatar
2. Tap the Upload button
3. See that the popup appears centered above the button
